packages/jest-config/src/__tests__/readInitialOptions.test.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import type {Config} from 'jest-runner';
```

Comment:
This is for `Config.InitialOptions`, right? If so, better import it directly from `@jest/types`:

```suggestion
import type {Config} from '@jest/types';
```